man/kernel-install.xml

```diff
@@ -321,6 +321,23 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--entry-type=type1|type2|all</option></term>
+
+        <listitem>
+          <para>
+            Controls the type of entries handled by the command. This is typically useful when multiple types
+            of kernels with the same version are installed, and removing one of the types.
```

Comment:
Is kernels the best word here, since a type2 entry will be a whole UKI? How about
```suggestion
            Controls the type of entries handled by the command. This is typically useful when multiple types
            of boot entries with the same kernel version are installed, and only one should beremoved.
```